pkg/operator/staticpod/startupmonitor/assets/startup-monitor-pod.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: {{.TargetNamespace}}
+  name: {{.TargetName}}-startup-monitor
+  labels:
+    revision: "REVISION"
+spec:
+  containers:
+    - name: startup-monitor
+      image: {{.OperatorImage}}
+      imagePullPolicy: IfNotPresent
+      terminationMessagePolicy: FallbackToLogsOnError
+      command: ["cluster-kube-apiserver-operator", "startup-monitor"] # todo: make it dynamic
+      args:
+        - {{.Verbosity}}
+        - --fallback-timeout-duration=300s
+        - --target-name=kube-apiserver
+        - --manifests-dir=/etc/kubernetes/manifests
+        - --resource-dir=/etc/kubernetes/static-pod-resources
+        - --revision=REVISION
+        - --kubeconfig=/etc/kubernetes/static-pod-resources/configmaps/kube-apiserver-cert-syncer-kubeconfig/kubeconfig #TODO: use a different config
+      volumeMounts:
+        - mountPath: /etc/kubernetes/manifests
+          name: manifests
+        - mountPath: /etc/kubernetes/static-pod-resources
+          name: resource-dir
+        - mountPath: /etc/kubernetes/static-pod-resources/secrets
+          subPath: secrets
+          name: pod-resource-dir
+          readOnly: true
+        - mountPath: /etc/kubernetes/static-pod-resources/configmaps
+          subPath: configmaps
+          name: pod-resource-dir
+          readOnly: true
+      resources:
+        requests:
+          memory: 50Mi
+          cpu: 5m
+      securityContext:
+        privileged: true
+  hostNetwork: true
+  terminationGracePeriodSeconds: 5
+  priorityClassName: system-node-critical
+  tolerations:
+    - operator: "Exists"
```

Comment:
do we need it?

Reply:
no idea. Better remove it instead of cargo culting :)